### PR TITLE
US 31 Admin Merchants Top Merchants Best Day

### PR DESCRIPTION
**Pull request by ttakahashi1591**
## Description
This PR enables a user to view the Top 5 Merchants best day listed below their information on the page. This PR includes the SQL and AR queries for the `best_day` method in the `Merchant` model. 

### Type of Change

- [ ] **fix**
- [x] **feat**
- [x] **test**
- [ ] **refactor**
- [ ] **docs**

### Checklist

- [x] **Code has been self reviewed**
- [x] **Code runs without any errors**
- [x] **Thorough testing has been implemented if adding feature**
- [x] **All tests pass**

### CODE REVIEW

File: spec/features/admin/merchants/index_spec.rb [hunk 1]
```diff
@@ -109,16 +109,108 @@
...
+        #   expect(page).to have_content(Merchant.best_day(merchant4))
+        # end
+
+        # within("#top_5-#{merchant5.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant5))
+        # end
+
+        # within("#top_5-#{merchant6.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant6))
+        # end
```
Comment (ttakahashi1591) [inline spec/features/admin/merchants/index_spec.rb:203]: @rjdelaguna I wanted to check in before removing these commented out lines. Let me know if I can remove them and I will commit it in this PR.

File: app/models/merchant.rb [hunk 2]
```diff
@@ -45,11 +45,36 @@ def items_ready_to_ship
...
+  #       inner join items on items.id = invoice_items.item_id
+  #       inner join merchants on merchants.id = items.merchant_id
+  #       inner join transactions on invoices.id = transactions.invoice_id
+  #     where merchants.id = #{merchant.id}
+  #     and transactions.result = 1
+  #     group by invoices.created_at
+  #     order by revenue desc
+  #     limit 1;"
+  #   )
+  # end
```
Comment (ttakahashi1591) [inline app/models/merchant.rb:61]: @deadbert I wanted to check in before removing these commented out lines. Let me know if I can remove them and I will commit it in this PR.

**Review by ttakahashi1591 — COMMENTED** [review]
Made these removed updates!

**Comment by ethanb1145** [comment]
Looks good! I like your use of flash messages to show the status update of being enabled or disabled.

**Pull request merged**